docs-2.0/7.data-security/3.manage-snapshot.md

```diff
@@ -0,0 +1,93 @@
+# Manage snapshots
+
+Nebula Graph supports using snapshots to backup and restore data.
+
+## Prerequisites
+
+If [authentication](1.authentication/1.authentication.md) is enabled, only the GOD-role user can use the snapshot function. For more information about roles, see [Roles and privileges](1.authentication/3.role-list.md).
+
+## Precautions
+
+* To prevent data loss, create a snapshot as soon as the system structure changes, for example, after operations such as `ADD HOST`, `DROP HOST`, `CREATE SPACE`, `DROP SPACE`, and `BALANCE` are performed.
+* Nebula Graph cannot automatically delete the invalid files created by a failed snapshot task, you have to manually delete them with [`DROP SNAPSHOT`](#delete_a_snapshot).
+* Customizing the storage path for the snapshots is not supported for now.
+
+## Snapshot form and path
+
+Nebula Graph snapshots are in the form of directories with names like `SNAPSHOT_2021_03_09_08_43_12`. The suffix `2021_03_09_08_43_12` is generated automatically based on the creation time.
+
+When a snapshot is created, snapshot directories will be automatically created in the `checkpoints` directory on the leader Meta server and each Storage server.
```

Comment:
what is the difference between Meta server and Meta Service? Is Meta server a new term?

Reply:
Meta Service is the service itself, held or run by nebula-metad processes. A Meta server is usually a server/machine that runs the nebula-metad processes. We can say that a directory is on a server/host/machine, but we can't say it is on a process.